core-web/libs/portlets/edit-ema/portlet/src/lib/shared/enums.ts

```diff
@@ -11,7 +11,8 @@ export enum NG_CUSTOM_EVENTS {
     CANCEL_SAVING_MENU_ORDER = 'cancel-save-menu-order',
     OPEN_WIZARD = 'workflow-wizard',
     DIALOG_CLOSED = 'dialog-closed',
-    EDIT_CONTENTLET_UPDATED = 'edit-contentlet-data-updated'
+    EDIT_CONTENTLET_UPDATED = 'edit-contentlet-data-updated',
+    URL_IS_CHANGED = 'url-is-changed'
```

Comment:
Do we need a new event for this? Can't we check if the URL changes in the `EDIT_CONTENTLET_UPDATED` event?

Reply:
I can send the data in that event, but I didn't want to change how that event works elsewhere (now only sent a boolean, no the data object), so I preferred to create another event

Reply:
Check the NgEvents handler in Shell component, we send the URL on Save, we can do the same we do there, there's no need for another event.